lib/framework/procbot-types.d.ts

```diff
@@ -54,14 +54,21 @@ export type SystemProperty = {
 	value: any;
 };
 
+export type PackageProperty = {
+	property: string;
+	value: any;
```

Comment:
This means that any nested keypair is only going to show up as `[object Object]` potentially in any output. Maybe not an issue, but to do it properly it should really be:
```typescript
value: PackageProperty | any;
```
along with relevant code to descend nested keypairs.

Reply:
And actually, I think it's more than that, I think it can be arrays of keypairs too, thinking about it.

Reply:
https://www.flowdock.com/app/rulemotion/testing/threads/qHdUgh9StUKyNsTeyHKY0pHl-4o

Reply:
Note to self: this feedback replaced with feedback to use util.inspect.

Reply:
PPS: done